dask/bag/core.py

```diff
@@ -846,6 +851,27 @@ def from_sequence(seq, partition_size=None, npartitions=None):
     return Bag(d, name, len(d))
 
 
+def from_url(urls):
+    """Create a dask.bag from a url
+
+    >>> a = from_url('http://google.com')
+    >>> a.npartitions
+    1
+
+    >>> b = from_url(['http://github.com', 'http://google.com'])
+    >>> b.npartions
+    2
+
+    """
+    if isinstance(urls, str):
+        urls = [urls]
+    dsk ={}
+    for i, u in enumerate(urls):
+        get_url = (urlopen, u)
+        dsk[('load-url', i)] = (lambda x: x.read(), get_url)
```

Comment:
Suggest 

    dsk[('load-url', i)] = (list, (urlopen, u))

urlopen probably gives back an iterator of lines.  We should take advantage of this laziness.  The list will be taken out if we can manage it.  This keeps things lazy and probably gives us better reaction speeds on large files.